src/PoolBuffer.cpp

```diff
@@ -84,13 +112,14 @@ PoolBuffer::PoolBuffer(int /*sample_rate*/, int channels, int bit_res, int FPP,
         mBitResolutionMode = AudioInterface::audioBitResolutionT::BIT32;
         break;
     }
-    mHist            = 6 * 32;                // samples, from original settings
+    mMinPoolSize     = mPoolSize;
```

Comment:
It looks like mMinPoolSize and mMaxPoolSize are superfluous now.

Reply:
yup, removed

Reply:
Great!